test/sql/function/timestamp/date_sub.test

```diff
@@ -0,0 +1,537 @@
+# name: test/sql/function/timestamp/date_sub.test
+# description: Test the DATESUB function
+# group: [timestamp]
+
+#
+# DATE
+#
+
+statement ok
```

Comment:
Could you add PRAGMA enable_verification?

Reply:
Copied and pasted from `date_diff.test`. I will fix it there too.